### PR TITLE
CF edits on the text, added the inset description, see also comment on the figure

### PR DESCRIPTION
**Pull request by ferrigno**
In the figure, I suggest to change the date format
from 
2022-10-09T13:17:00
to 
2022-10-09 13:17:00
(remove the "T")

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,14 @@
...
 
-
-
-The image shows GRB 221009A lightcurve recorded by several INTEGRAL instruments. On the left - the prompt emission phase as measured by SPI-ACS, showing the brightest part of the GRB and an extended tail lasting few hours, detected even by a non-imaging detector. On the right - three revolutions of ISGRI and JEM-X observations of the hard X-ray afterglow. For the last point, only JEM-X yields a significant detection. 
+The image shows the GRB 221009A lightcurve recorded by several INTEGRAL instruments. On the lefti, before 0.1 s, there is the prompt emission phase as measured by SPI-ACS, showing the brightest part of the GRB and an extended tail lasting few hours, detected even by a non-imaging detector. On the right, afte 0.1 s, we show three revolutions of IBIS/ISGRI and JEM-X observations of the hard X-ray afterglow. For the last point, only JEM-X yields a significant detection. The inset shows the IBIS/ISGRI image of the afterglow.
```
Comment (volodymyrss) [inline README.md:12]: it's not 0.1 seconds, obviously. It's also written on the picture

File: README.md [hunk 2]
```diff
@@ -2,16 +2,14 @@
...
 
-The intensity of the GRB was so high, that its afterglow was detected in INTEGRAL/JEM-X almost during a week, far exceeding the previous record of a few hours. Long-lasting afterglows can also appear without prompt GRB detection in case the collimated source (jet) is not pointing straight to us (as it was also the case in GW170817). However it can be detected at a later stage when the jet decelerates and its opening angle widens. In the future, hard X-ray follow-ups of rapidly expanding population of Gravitational Waves (GW) sources will possibly reveal these so-called "orphan afterglows", providing precise localizations for sources located at distances which will be probed by the GW detectors in the coming years (z<~0.1).
```
Comment (volodymyrss) [inline README.md:10]: the suggestion of Diego was actually to replace for with during. These sort of style changes are only so much useful anyway

File: README.md (hunk 1 above)
Comment (volodymyrss) [inline README.md:12]: I already added it actually, so I keep my version.

**Comment by volodymyrss** [comment]
I incorporate outside of the PR.

File: README.md (hunk 1 above)
Comment (ferrigno) [reply README.md:12]: 0.1 days, yes.

**Pull request closed without merging**